components/setup/BuildInstruction.tsx

```diff
@@ -43,9 +43,9 @@ const BuildInstruction: React.FC<BuildInstructionProps> = ({ selectedHosting, us
     };
 
 	const encrypt_github_pat = (handleGithubPatInputValue: string) => {
-		let secret_key = process.env.NEXT_PUBLIC_SECRET_KEY;
-		if (secret_key) {
-			return encrypt(secret_key, handleGithubPatInputValue);
+		let github_pat_encryption_secret_key = process.env.NEXT_PUBLIC_GITHUB_PAT_ENCRYPTION_SECRET_KEY;
+		if (github_pat_encryption_secret_key) {
```

Comment:
@tapishr Does it make more sense to encrypt on the client side or the server side?

Client side: PAT will be encrypted in transit over the network. But the key to decrypt it is publicly available for anyone using the website. Especially because this secret is static across users.

Server side: For attackers sniffing on the network, they just get the PAT unencrypted, but the key to the encryption is only known to us. If a data breach happens on our end, there is no way that the attacker can get the key without social engineering or hacking Google.